server/sonar-server/src/main/resources/org/sonar/server/computation/ws/quality_gate_status-example.json

```diff
@@ -0,0 +1,70 @@
+{
+  "qualityGateDetails": {
+    "level": "ERROR",
+    "conditions": [
+      {
+        "metric": "new_coverage",
+        "op": "LT",
+        "period": "1",
+        "warning": "",
```

Comment:
whatever the level of information you need (as I said in any case some info are missing for the build breaker), database internal format must never be exposed.